plugins/inputs/nfsclient/nfsclient.go

```diff
@@ -33,20 +33,19 @@ type NFSClient struct {
 func convertToUint64(line []string) ([]uint64, error) {
 	/* A "line" of input data (a pre-split array of strings) is
 	   processed one field at a time.  Each field is converted to
-	   an uint64 value, and appened to an array of return values.
+	   an uint64 value, and appended to an array of return values.
 	   On an error, check for ErrRange, and returns an error
 	   if found.  This situation indicates a pretty major issue in
 	   the /proc/self/mountstats file, and returning faulty data
 	   is worse than no data.  Other errors are ignored, and append
 	   whatever we got in the first place (probably 0).
 	   Yes, this is ugly. */
 
-	var nline []uint64
-
 	if len(line) < 2 {
-		return nline, nil
+		return nil, nil
```

Comment:
hmm where this is called will throw an error as it immediately calls `len()` on the return value and `len(nil)` won't work, right?

Reply:
`len` should return for `nil` slice

https://go.dev/play/p/U0v-Gk_5R11

Reply:
ah right